optimism/src/lookup.rs

```diff
@@ -114,3 +114,27 @@ pub trait Lookups {
     /// Adds all lookups of Self to the environment
     fn lookups(&mut self);
 }
+
+/// A table of values that can be used for a lookup, along with the ID for the table.
+#[derive(Debug, Clone)]
+pub struct LookupTable<F> {
+    /// The table is a vector of write lookups with the same table ID
+    _table: Vec<Lookup<F>>,
```

Comment:
`lookup_requests` or `lookup_values`?

Reply:
requests would make sense for `read` types of entries, but these are (for now) always `write` lookups (meaning, they already exist and we will just consume a few of them throughout the circuit execution).